addon/components/lazy-render-wrapper.js

```diff
@@ -120,46 +120,72 @@ export default Ember.Component.extend({
 	_shouldShowOnRender: false,
 
 	event: 'hover', // hover, click, focus, none
-	entryInteractionEvents: computed('event', function() {
+	_lazyRenderEvents: computed('event', function() {
 		// the lazy-render wrapper will only render the tooltip when
-		// the $parent element is interacted with. This CP defines which
-		// events will trigger the rendering. We always include focusin
-		// to keep the component accessible.
-		let entryInteractionEvents = ['focusin'];
+		// the $target element is interacted with. This CP defines which
+		// events will trigger the rendering. Unless event="none" we always
+		// include focusin to keep the component accessible.
 		let event = this.get('event');
+
+		if (event === 'none') {
+			return [];
+		}
+
+		let _lazyRenderEvents = ['focusin'];
+
 		if (event === 'hover') {
-			entryInteractionEvents.push('mouseenter');
+			_lazyRenderEvents.push('mouseenter');
 		} else if (event === 'click') {
-			entryInteractionEvents.push('click');
+			_lazyRenderEvents.push('click');
+		}
+
+		return _lazyRenderEvents;
+	}),
+
+	$target: computed('target', 'tetherComponentName', function() {
```

Comment:
Can you add a bit of documentation to describe why this CP is necessary? I like the style of the [YUI Docs Syntax](http://yui.github.io/yuidoc/syntax/index.html) and have tried to use that in a couple of places. What do you think?